website/docs/d/vra_region_enumeration_gcp.html.markdown

```diff
@@ -0,0 +1,31 @@
+---layout: "vra"
+page_title: "VMware vRealize Automation: vra_region_enumeration_gcp"
+description: |-
+  Provides a data lookup for vra_region_enumeration_gcp.
+---
+
+# Data Source: vra_region_enumeration_gcp
+## Example Usages
+
+This is an example of how to read a region enumeration data source for GCP cloud account.
+
+**Region enumeration GCP data source by its id:**
+```hcl
+data "vra_region_enumeration_gcp" "this" {
+  id = vra_region_enumeration_gcp.this.id
+}
+```
+## Argument Reference
+
+* `id` - (Optional) The id of the region enumeration for GCP account.
+
+## Attribute Reference
+
+* `private_key` - GCP Private key.
+
+* `private_key_id` - GCP Private key ID.
+
+* `project_id` - GCP Project ID.
```

Comment:
Same comment as above. `id` is attribute. `client_email`, `private_key`, `private_key_id`, and `project_id` are `Required` arguments.

Correct the example as well please.